commands/chatneighbour/index.js

```diff
@@ -41,16 +41,13 @@ module.exports = {
 	
 		const index = list.indexOf(userData.Name);
 		if (index === -1) {
-			return {
-				success: false,
-				reply: "That user is not currently present in chat!"
-			};
+			list.push(userData.Name);
```

Comment:
This should only happen when it is the user who invoked the command. If it's someone else, the command should exit as before. Also, after pushing the username, the array must be re-soreted and the index re-calculated.